src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -61,6 +61,9 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
     socket.onmessage = (event: MessageEvent) => {
       try {
         const json = JSON.parse(event.data);
+        if (json?.message?.message_type === 'text') {
+          json.message.attachments = [];
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change could potentially lead to data loss. If the message type is 'text', any existing attachments are being cleared. This might not be the desired behavior if there are important attachments associated with the text messages. Consider preserving the attachments or handling them in a different way.
```diff
-          json.message.attachments = [];
+          // Handle attachments for 'text' message type appropriately
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->